### ISSUE TITLE
Tracking issue for BitChute

### ISSUE THREAD

**Issue by calzoneman** [calzoneman/sync#928]
```
2019-10-18.log:[09:55:16] <Xaekai> The total requests I've gotten for bitchute support is up to 43
```

I guess the total is at least 44 now since I just got emailed another one.

Previously, requests for bitchute could not be actioned because the site lacked a usable API for synchronized embedding.  I'm not sure if that is still the case, but I'm filing this issue anyways so there is at least something to link people who ask.

**Comment by Xaekai** [comment]
They still lack an API, and are likely to continue to lack an API. Somehow they manage to pull in $30,000 a month and yet seem to do nothing. (They've been around ~5 years and most videos seem to only support a single resolution? wtf?)

yt-dlp can easily give an API like info block in a readily consumable JSON format (`yt-dlp -j %url%`) with a direct mp4 link. It lacks duration, but the direct link can be ffprobed. Given the demand for this site, the unlikelyhood of them adding an API and the unlikelyhood of them disappearing, I believe this is the course of action we should take.

**Comment by Xaekai** [comment]
Resolved in pending CyTube 3.90-rc release.
